internal/services/steelthreadtesting/steelthread_test.go

```diff
@@ -0,0 +1,75 @@
+//go:build steelthread
+// +build steelthread
+
+package steelthreadtesting
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	yamlv3 "gopkg.in/yaml.v3"
+
+	v1 "github.com/authzed/authzed-go/proto/authzed/api/v1"
+	"github.com/authzed/spicedb/internal/testserver"
+	"github.com/authzed/spicedb/pkg/datastore"
+	"github.com/authzed/spicedb/pkg/validationfile"
+	"github.com/dustin/go-humanize"
+	"github.com/gosimple/slug"
+	"github.com/stretchr/testify/require"
+)
+
+const defaultConnBufferSize = humanize.MiByte
+
+func TestSteelThreads(t *testing.T) {
+	for _, tc := range steelThreadTestCases {
+		t.Run(tc.name, func(t *testing.T) {
+			req := require.New(t)
```

Comment:
@josephschorr any reason steel threads are not running in parallel?

Reply:
No, I just didn't do so in the initial PRs